docs/core/diagnostics/runtime-contention-events.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Thread Contention Events"
+description: See ETW events that collect information specific to the managed thread contention.
+ms.date: "09/27/2020"
+ms.topic: reference
+helpviewer_keywords:
+  - "thread contention events" [.NET Core]"
+  - "ETW, EventPipe, LTTng contention events (CoreCLR)"
+ms.assetid: 167a4459-bb6e-476c-9046-7920880f2bb5
+---
+
+# Contention Events
+
+These runtime events capture information about managed thread contentions.
```

Comment:
The contention events are specific to locks taken with `Monitor` (or the `lock` keyword in C#). Maybe change `managed thread contentions` to `Monitor lock contentions` everywhere in this file, and maybe change this instance to `Monitor lock contentions such as with Monitor.Enter or the C# lock keyword`.